src/akshita/Chracter.java

```diff
@@ -0,0 +1,38 @@
+//WAP to find count of Upper case, Lower case,Digits and Special Character
+//input: TecHNo_cR6ediTs
+//Output: Upper case:5, Lower case-8, Digit-1, Special char-1
+
+package akshita;
+
+public class Chracter {
+
+	public static void main(String[] args) {
+        int Ucount=0;
+        int Lcount=0;
+        int Dcount=0;
+        int Scount=0;
+		String var = "TecHNo_cR6ediTs";
+			for(int index=0;index<var.length();index++) {
+				 if(Character.isUpperCase(var.charAt(index))) {
```

Comment:
you can write char ch = var.charAt(index) outside of conditional statement, so that you no need to rewrite again and again. you just need to write like Character.isUpperCase(ch)